### ISSUE TITLE
Desktop & Mobile Menus at same time

### ISSUE THREAD

**Issue by JourneysCT** [pojome/elementor#6404]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [ ] The issue still exists against the latest stable version of Elementor.

## Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a default WordPress theme active.
- [ ] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>
					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: Percona Server (GPL), Release 84.1, Revision b308619 v5.6.41-84.1
	PHP Version: 7.0.31
	PHP Max Input Vars: 1000
	PHP Max Post Size: 64M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.0
	Site URL: https://journeysgp.walnutcreekhoa.com
	Home URL: https://journeysgp.walnutcreekhoa.com
	WP Multisite: No
	Max Upload Size: 64 MB
	Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: America/Denver
	Debug Mode: Inactive

== Theme ==
	Name: GeneratePress Child
	Version: 0.1
	Author: Tom Usborne
	Child Theme: Yes
	Parent Theme Name: GeneratePress
	Parent Theme Version: 2.2.1
	Parent Theme Author: Tom Usborne

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.80 Safari/537.36

== Active Plugins ==
	All-in-One WP Migration
		Version: 6.81
		Author: ServMask

	Central Color Palette
		Version: 1.12.6
		Author: Gáravo

	Code Snippets
		Version: 2.12.1
		Author: Shea Bunge

	CSS Hero
		Version: 3.41
		Author: The CSS Hero Team

	Duplicate Post
		Version: 3.2.2
		Author: Enrico Battocchi

	Easy Theme and Plugin Upgrades
		Version: 2.0.1
		Author: Chris Jean

	Elementor
		Version: 2.3.5
		Author: Elementor.com

	Elementor Pro
		Version: 2.2.5
		Author: Elementor.com

	Email Address Encoder
		Version: 1.0.12
		Author: Till Krüss

	Envato Elements - Template Kits (Beta)
		Version: 0.1.2
		Author: Envato

	Flexible Elementor Panel
		Version: 1.8.1
		Author: Creative Web Solution

	GP Premium
		Version: 1.7.5
		Author: Tom Usborne

	JetBlocks For Elementor
		Version: 1.1.4
		Author: Zemez

	JetBlog For Elementor
		Version: 2.1.8
		Author: Zemez

	JetElements Dynamic Data Addon
		Version: 1.1.0
		Author: Zemez

	Jet Elements For Elementor
		Version: 1.14.6
		Author: Zemez

	Jet Plugins Wizard
		Version: 1.2.1
		Author: Zemez

	Jet Popup
		Version: 1.2.3
		Author: Zemez

	JetTabs For Elementor
		Version: 1.1.4
		Author: Zemez

	JetThemeCore
		Version: 1.1.6
		Author: Zemez

	JetThemeCore Compatibility Kit
		Version: 1.1.0
		Author: Zemez

	Simple CSS
		Version: 1.1
		Author: Tom Usborne

	TRO Brochure Rack
		Version: 1.0
		Author: Travel Research Online

	UpdraftPlus - Backup/Restore
		Version: 1.15.7
		Author: UpdraftPlus.Com, DavidAnderson

	Wordfence Security
		Version: 7.1.18
		Author: Wordfence

== Must-Use Plugins ==
	SSO
		Version: 0.3
		Author: Garth Mortensen, Mike Hansen

== Debug ==
	Errors: 
		ReferenceError: Can't find variable: google
			at https://journeysgp.walnutcreekhoa.com/wp-content/plugins/premium-addons-for-elementor/assets/js/premium-maps.js?ver=2.6.4 - 43:33
			2018-09-17 04:41 +00:00
			x 2 times

		ReferenceError: Can't find variable: google
			at https://journeysgp.walnutcreekhoa.com/wp-content/plugins/premium-addons-for-elementor/assets/js/premium-maps.js?ver=2.6.4 - 43:33
			2018-09-17 03:50 +00:00
			x 2 times

		ReferenceError: Can't find variable: google
			at https://journeysgp.walnutcreekhoa.com/wp-content/plugins/premium-addons-for-elementor/assets/js/premium-maps.js?ver=2.5.7 - 43:33
			2018-08-21 20:51 +00:00
			x 3 times

		ReferenceError: google is not defined
			at https://journeysgp.walnutcreekhoa.com/wp-content/plugins/premium-addons-for-elementor/assets/js/premium-maps.js?ver=2.5.4 - 30:23
			2018-08-08 20:53 +00:00
			x 3 times

		TypeError: t.stick_in_parent is not a function
			at https://journeysgp.walnutcreekhoa.com/wp-content/plugins/elementor-pro/assets/js/frontend.min.js - 2:34780
			2018-07-16 18:11 +00:00

		TypeError: undefined is not an object (evaluating 'this.ui.tooltipTarget.tipsy')
			at https://journeysgp.walnutcreekhoa.com/wp-content/plugins/elementor/assets/js/editor.min.js - 2:50872
			2018-06-26 20:54 +00:00
			x 2 times

		TypeError: e is undefined
			at https://journeysgp.walnutcreekhoa.com/wp-content/plugins/elementor/assets/js/editor.min.js - 2:171439
			2018-06-26 08:29 +00:00
			x 2 times

		TypeError: e is undefined
			at https://journeysgp.walnutcreekhoa.com/wp-content/plugins/elementor/assets/js/editor.min.js - 2:117962
			2018-06-20 20:50 +00:00
			x 2 times

		TypeError: undefined is not an object (evaluating 'this._parent.errors')
			at https://journeysgp.walnutcreekhoa.com/wp-content/plugins/elementor/assets/js/editor.min.js - 2:132426
			2018-05-21 05:21 +00:00
			x 19 times

		TypeError: undefined is not an object (evaluating 'this._parent.errors')
			at https://journeysgp.walnutcreekhoa.com/wp-content/plugins/elementor/assets/js/editor.min.js - 2:132925
			2018-05-01 00:20 +00:00

				
I installed the 2.3.0 beta-1 update and immediately I am getting both the Desktop and Mobile Menus appearing across all devices. I did revert back to 2.2.5 and the issue is resolved. I'm not sure how else to say this, so hopefully that is sufficient. If not, please advise as to what I can do to help further. Thank You!

**Comment by CharfishCharlie** [comment]
Same for me. Using GeneratePress Pro and Elementor Pro. 

GeneratePress options have all menu options turned off and I'm using an Elementor Header Template which overrides that functionality anyway.

![screen shot 2018-12-11 at 9 09 32 pm](https://user-images.githubusercontent.com/1186866/49848274-42ed6080-fd89-11e8-89b9-da337f469ba4.png)

**Comment by bainternet** [comment]
Fixed in 2.3.0 thanks.
